sos/plugins/openvswitch.py

```diff
@@ -42,6 +43,21 @@ def setup(self):
             "/var/run/openvswitch/ovsdb-server.pid",
             "/var/run/openvswitch/ovs-vswitchd.pid"
         ])
+        # Collect ovs-vswitchd's pmd thread details in procfs.
+        pid = open("/var/run/openvswitch/ovs-vswitchd.pid").read()\
+            .splitlines()[0]
+        # List TID & thread command name.
+        ps_tidlist_cmd = "ps -p %s -Lo tid=,comm= " % pid
+        tid_list = filter(
+                re.compile(r'pmd').search,
+                self.call_ext_prog(ps_tidlist_cmd)['output'].splitlines())
+        # filter out list elements with string 'pmd' & if not empty,
+        # capture process details.
+        if tid_list:
```

Comment:
IMHO redundant test since you gate just `for tid in tid_list` cycle by this test.